content/blog/example-of-mapreduce-with-ezmeral-data-fabric-mapr-database-binary-table.md

```diff
@@ -0,0 +1,365 @@
+                DISK_MILLIS_MAPS=2816
+                DISK_MILLIS_REDUCES=3930
+        Map-Reduce Framework
+                Map input records=4
+                Map output records=4
+                Map output bytes=55
+                Map output materialized bytes=0
+                Input split bytes=181
+                Combine input records=0
+                Combine output records=0
+                Reduce input groups=2
+                Reduce shuffle bytes=65
+                Reduce input records=4
+                Reduce output records=2
+                Spilled Records=8
+                Shuffled Maps =1
+                Failed Shuffles=0
+                Merged Map outputs=2
+                GC time elapsed (ms)=209
+                CPU time spent (ms)=4600
+                Physical memory (bytes) snapshot=1240059904
+                Virtual memory (bytes) snapshot=9576202240
+                Total committed heap usage (bytes)=1421869056
+        HBase Counters
+                BYTES_IN_REMOTE_RESULTS=0
+                BYTES_IN_RESULTS=0
+                MILLIS_BETWEEN_NEXTS=0
+                NOT_SERVING_REGION_EXCEPTION=0
+                NUM_SCANNER_RESTARTS=0
+                NUM_SCAN_RESULTS_STALE=0
+                REGIONS_SCANNED=0
+                REMOTE_RPC_CALLS=0
+                REMOTE_RPC_RETRIES=0
+                ROWS_FILTERED=0
+                ROWS_SCANNED=0
+                RPC_CALLS=0
+                RPC_RETRIES=0
+        Shuffle Errors
+                IO_ERROR=0
+        File Input Format Counters
+                Bytes Read=0
+        File Output Format Counters
+                Bytes Written=0
+
+```
+
+
+
+**Important note**: If you encountered the issue that some of the HBase related Java packages are missing, you can simply copy the following packages from 📁<ins>/opt/mapr/hbase/hbase-{VERSION}/lib/</ins> to 📁<ins>/opt/mapr/hadoop/hadoop-{VERSION}/share/hadoop/common/</ins>.
+
+
+
+- hbase-client-\*.jar
+- hbase-server-\*.jar
+- hbase-protocol-\*.jar
+- hbase-hadoop2-compat-\*.jar
+- hbase-hadoop-compat-\*.jar
+- hbase-metrics-\*.jar
+- hbase-metrics-api-\*.jar
+- hbase-shaded-gson-\*.jar
+- hbase-shaded-htrace-\*.jar
+- metrics-core-\*.jar
+- hbase-common-\*.jar
+
+
+
+This is because the HBase related packages are considered as third-party libraries to the Hadoop system, refer to👉: [Install the third-party libraries on each node that runs the program][InstallTheThird-partyLibrariesOn].
+
+
+
+## Explanation of some glossary
+
+
+
+<details>
+<summary>HPE Ezmeral Data Fabric (AKA. MapR)</summary>
+
+
+
+EDF for short.
+HPE Ezmeral Data Fabric is a platform for data-driven analytics, ML, and AI workloads.
+The platform serves as a secure data store and provides file storage, NoSQL databases, object storage, and event streams.
+The patented filesystem architecture was designed and built for performance, reliability, and scalability.
+📖[Documentation website][EDFDocumentation]
+
+

```

Comment:
You need a summary to this blog post. It doesn't have to be very long. It's just a little closing saying something like "I hope you found this blog post helpful in... (and then redescribe what it does). This can prove extremely helpful in situations like (and describe a few). Stay tuned to the HPE Developer blog for more interesting posts and tutorials on HPE Ezmeral Data Fabric. [and then hotlink "HPE Developer blog" to https://developer.hpe.com/blog .